src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -81,18 +81,20 @@ class RtcpRrGenerator(
             val streamStats = incomingStatisticsTracker.getSnapshot()
             val now = System.currentTimeMillis()
             val reportBlocks = mutableListOf<RtcpReportBlock>()
-            streamStats.ssrcStats.forEach { ssrc, statsSnapshot ->
+            streamStats.ssrcStats.forEach { (ssrc, statsSnapshot) ->
                 val senderInfo = senderInfos.computeIfAbsent(ssrc) {
                     SenderInfo()
                 }
-                val statsDelta = statsSnapshot.getDelta(senderInfo.statsSnapshot)
+                val fractionLost = statsSnapshot.computeFractionLost(senderInfo.statsSnapshot)
+                senderInfo.statsSnapshot = statsSnapshot
+
                 reportBlocks.add(RtcpReportBlock(
                     ssrc,
-                    statsDelta.fractionLost,
-                    statsDelta.cumulativePacketsLost,
-                    statsDelta.seqNumCycles,
-                    statsDelta.maxSeqNum,
-                    statsDelta.jitter.toLong(),
+                    fractionLost,
+                    statsSnapshot.cumulativePacketsLost,
```

Comment:
are these values not supposed to be deltas from the last snapshot/RR? i thought that's what was expected in an RR

Reply:
fraction lost is for the interval between two rrs, but cumulative packets lost is from the beginning of reception